using-timescaledb/continuous-aggregates.md

```diff
@@ -96,6 +96,120 @@ with PostgreSQL. This means that aggregates with `ORDER BY` clauses or `DISTINCT
+the hypertable using
+[`set_integer_now_func`][api-set-integer-now-func]. The function can
+be defined as a normal Postgres function, but need to be
+[`STABLE`][pg-func-stable], take no arguments, and should return an
+integer of the same type as the time column in the table. In our case,
+this should suffice:
+
+```sql
+CREATE OR REPLACE FUNCTION current_microfortnight() RETURNS BIGINT
+LANGUAGE SQL STABLE AS $$
+	SELECT CAST(1209600 * EXTRACT(EPOCH FROM CURRENT_TIMESTAMP) / 1000000 AS BIGINT)
+$$;
+
+SELECT set_integer_now_func('devices', 'current_microfortnight');
+```
+
+Once the replacement for current timestamp has been set up, you can
+define the continuous aggregate for the `devices` table.
+
+```sql
+CREATE VIEW devices_summary
+WITH (
+  timescaledb.continuous,
+  timescaledb.refresh_interval='1 minutes',
+  timescaledb.refresh_lag = '-5')
+AS SELECT time_bucket('5', time) AS bucket,
+	      avg(cpu_usage) AS avg_cpu,
+	      avg(disk_usage) AS avg_disk
+     FROM devices
+    GROUP BY bucket;
+```
+
+You can now insert some rows to check if the aggregation works as
+expected. In this case, we are using the `tablefunc` extension to
+generate a Gaussian distribution of CPU usage and a cumulative growth
+of disk usage.
+
+```sql
+INSERT INTO devices(time, cpu_usage, disk_usage)
+SELECT time,
+       normal_rand(1,70,10) AS cpu_usage,
+	   normal_rand(1,2,1) * (row_number() over()) AS disk_usage
+  FROM generate_series(1,10000) AS time;
+```
+
+If you wait one minute, the view will be refreshed and you will be
+able to check the contents using:
+
+```sql
+postgres=# SELECT * FROM devices_summary ORDER BY bucket;
+ bucket |       avg_cpu       |       avg_disk
+--------+---------------------+----------------------
+      0 | 63.0000000000000000 |   6.0000000000000000
+      5 | 69.8000000000000000 |   9.6000000000000000
+     10 | 70.8000000000000000 |  24.0000000000000000
+     15 | 75.8000000000000000 |  37.6000000000000000
+     20 | 71.6000000000000000 |  26.8000000000000000
+     25 | 67.6000000000000000 |  56.0000000000000000
+     30 | 68.8000000000000000 |  90.2000000000000000
+     35 | 71.6000000000000000 |  88.8000000000000000
+     40 | 66.4000000000000000 |  81.2000000000000000
+     45 | 68.2000000000000000 | 106.0000000000000000
+     50 | 72.2000000000000000 | 105.0000000000000000
+     55 | 66.2000000000000000 | 138.8000000000000000
+     60 | 61.6000000000000000 | 131.4000000000000000
+     65 | 64.2000000000000000 | 136.4000000000000000
+     70 | 72.0000000000000000 | 114.8000000000000000
+     75 | 64.4000000000000000 | 155.2000000000000000
+     80 | 65.8000000000000000 | 175.2000000000000000
+     85 | 68.0000000000000000 | 156.8000000000000000
+     90 | 71.6000000000000000 | 151.4000000000000000
+     95 | 68.6000000000000000 | 136.2000000000000000
+(20 rows)
+```
+
+If you want to refresh the view immediately, you can do it in the
+normal manner:
+
+```sql
+REFRESH MATERIALIZED VIEW devices_summary;
+```
```

Comment:
@gayyappan When I do this, I get the following errors:

```
postgres=# refresh materialized view devices_summary;
INFO:  new materialization range for public.devices (time column time) larger than allowed in one run, truncating 1915095555 to 100
INFO:  materializing continuous aggregate public.devices_summary: nothing to invalidate, new range up to 100
WARNING:  REFRESH did not materialize the entire range since it was limited by the max_interval_per_job setting
REFRESH MATERIALIZED VIEW
```

Which seems mostly confusing and users are guaranteed to react to it and wonder what they have done wrong. Shall we add a notice to the documentation and/or change the error message?